meta/plenta/contao-jobs-basic-zvoove-bundle/de.yml

```diff
@@ -0,0 +1,24 @@
+de:
+    title: zvoove Connector für Contao Jobs Basic
+    description: >
+        Stellenanzeigen werden regelmäßig von zvoove importiert, während Bewerbungen direkt aus Contao an die entsprechende Stellenanzeige in Zvoove gesendet werden.
```

Comment:
```suggestion
        Stellenanzeigen werden regelmäßig von zvoove importiert, während Bewerbungen direkt aus Contao an die entsprechende Stellenanzeige in zvoove gesendet werden.
```

Reply:
done @fritzmg

Reply:
@christianbarkowsky what do you mean here?